### PR TITLE
Portfolio: First Project Added.

### PR DESCRIPTION
**Pull request by ezraermy**
This is a pull request. I:
- created branches with different base branches.
- Followed GitHub flow.
- Added a descriptive and easy to understand README
- Added a HTML file
- Added a CSS File

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -24,15 +24,45 @@ <h1 class="title">Hi! I am Ezra nice to meet you.</h1>
...
+        <p class="highlights">
+            A daily selection of privately personalized reads
+            no accounts for sign-ups required.
+        </p>
+        <ul class="tags">
+            <li><button>HTML</button></li>
+            <li><button>CSS</button></li>
+            <li><button>Js</button></li>
+        </ul>
+        <button class="project-detail">See <br> Project</button>
```
Comment (ghost) [inline index.html]: - Kindly work on the padding of your `see project `  button to match the Figma design

image

### Checkout Figma

image

File: index.html [hunk 2]
```diff
@@ -24,15 +24,45 @@ <h1 class="title">Hi! I am Ezra nice to meet you.</h1>
...
+    <section class="work-page">
+        <img src="images/works/portfolio1.svg" alt="work1" width="100%">
+        <h2 class="work-title">Tonic</h2>
+        <ul class="clients">
+            <li>CANOPY</li>
+            <img src="images/works/Counter.svg" alt="counter" class="counter">
+            <li>BackEndDev</li>
+            <img src="images/works/Counter.svg" alt="counter" class="counter">
+            <li>2015</li>
+        </ul>
```
Comment (ghost) [inline index.html]: - [ ] Kindly space these elements correctly to match the Figma design
  
image

### Checkout Figma
image
 design

- [ ] I noticed you only had one project card. Kindly add three more project cards to match the Figma design

File: index.html [hunk 3]
```diff
@@ -24,15 +24,45 @@ <h1 class="title">Hi! I am Ezra nice to meet you.</h1>
...
+            no accounts for sign-ups required.
+        </p>
+        <ul class="tags">
+            <li><button>HTML</button></li>
+            <li><button>CSS</button></li>
+            <li><button>Js</button></li>
+        </ul>
+        <button class="project-detail">See <br> Project</button>
+    </section>
+    <section id="about-myself">
```
Comment (ghost) [inline index.html]: - [ ] Kindly work on the spacing of elements in you `about me` section to match the Figma design.

image

### Checkout Figma

image

- [ ] There are some sections missing in your `about me`  section. Kindly add them.

image

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @ezraermy , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -24,15 +24,118 @@ <h1 class="title">Hi! I am Ezra nice to meet you.</h1>
...
             <ul class="socials">
                 <li><a href="#" class="fa fa-facebook"></a></li>
                 <li><a href="#" class="fa fa-twitter"></a></li>
                 <li><a href="#" class="fa fa-linkedin"></a></li>
                 <li><a href="#" class="fa fa-google"></a></li>
                 <li><a href="#" class="fa fa-github"></a></li>
+                <li><a href="#" class="fa fa-javascript"></a></li>
             </ul>
-        </main>
+        <button class="resume">Get My Resume</button>
```
Comment (victorokoroji) [inline index.html:123]: - [ ] Kindly change the background color of all your buttons  and the border around the button in the about section to match the Figma design 👍🏽 

FIgma Design 👇🏽 
image

YOur Design 👇🏽 
image

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @ezraermy, 

Great work on making the changes requested by a previous reviewer 👏 🔥 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



## Top highlight 🎯

- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (ezraermy) [reply index.html:123]: Thanks for the tips. Correction made.

File: index.html [hunk 5]
```diff
@@ -24,15 +24,118 @@ <h1 class="title">Hi! I am Ezra nice to meet you.</h1>
...
+        <button class="project-detail">See Project</button>
+    </section>
+
+    <!-- Second project -->
+    <section class="work-page">
+        <img src="images/works/multi_post_story/cover.svg" alt="work1" width="100%">
+        <h2 class="work-title">Tonic</h2>
+        <ul class="clients">
+            <li>CANOPY</li>&nbsp;
+            <img src="images/works/Counter.svg" alt="counter" class="counter">&nbsp;
```
Comment (bienvenuushindi) [inline index.html:61]: - [ ] **OPTIONAL**  :warning: Only li tag is allowed as a child of ul. Please make sure you wrap the img tag inside a li tag

File: index.html [hunk 6]
```diff
@@ -24,15 +24,118 @@ <h1 class="title">Hi! I am Ezra nice to meet you.</h1>
...
+            <li>Back</li>&nbsp;
+            <li>End</li>&nbsp;
+            <li>Dev</li>&nbsp;
+            <img src="images/works/Counter.svg" alt="counter" class="counter">&nbsp;
```
Comment (bienvenuushindi) [inline index.html:65]: - [ ] **OPTIONAL**  :warning: Only li tag is allowed as a child of ul. Please make sure you wrap the img tag inside a li tag

File: index.html [hunk 7]
```diff
@@ -24,15 +24,118 @@ <h1 class="title">Hi! I am Ezra nice to meet you.</h1>
...
+        </main>
+    </section>
+    
+    <!-- First Project -->
+    <section class="work-page">
+        <img src="images/works/tonic/cover1.svg" alt="work1" width="100%">
+        <h2 class="work-title">Tonic</h2>
+        <ul class="clients">
+            <li>CANOPY</li>&nbsp;
+            <img src="images/works/Counter.svg" alt="counter" class="counter">&nbsp;
```
Comment (bienvenuushindi) [inline index.html:36]: - [ ] **OPTIONAL**  :warning: Only li tag is allowed as a child of ul. Please make sure you wrap the img tag inside a li tag

File: index.html [hunk 8]
```diff
@@ -24,15 +24,118 @@ <h1 class="title">Hi! I am Ezra nice to meet you.</h1>
...
+            <li>Back</li>&nbsp;
+            <li>End</li>&nbsp;
+            <li>Dev</li>&nbsp;
+            <img src="images/works/Counter.svg" alt="counter" class="counter">&nbsp;
```
Comment (bienvenuushindi) [inline index.html:40]: - [ ] **OPTIONAL**  :warning: Only li tag is allowed as a child of ul. Please make sure you wrap the img tag inside a li tag

File: index.html [hunk 9]
```diff
@@ -24,15 +24,118 @@ <h1 class="title">Hi! I am Ezra nice to meet you.</h1>
...
```
Comment (bienvenuushindi) [inline index.html:90]: - [ ] **OPTIONAL**  :warning: Only li tag is allowed as a child of ul. Please make sure you wrap the img tag inside a li tag

File: index.html [hunk 10]
```diff
@@ -24,15 +24,118 @@ <h1 class="title">Hi! I am Ezra nice to meet you.</h1>
...
+        <button class="project-detail">See Project</button>
+
+    <!-- Third project -->
+    </section>
+    <section class="work-page">
+        <img src="images/works/tonic/cover1.svg" alt="work1" width="100%">
+        <h2 class="work-title">Tonic</h2>
+        <ul class="clients">
+            <li>CANOPY</li>&nbsp;
+            <img src="images/works/Counter.svg" alt="counter" class="counter">&nbsp;
```
Comment (bienvenuushindi) [inline index.html:86]: - [ ] **OPTIONAL**  :warning: Only li tag is allowed as a child of ul. Please make sure you wrap the img tag inside a li tag

File: style.css [hunk 11]
```diff
@@ -67,28 +72,239 @@ section {
...
+  right: 78px;
+  top: calc(50% - 24px / 2);
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  color: #000;
 }
```
Comment (bienvenuushindi) [inline style.css:310]: - [ ] **OPTIONAL** 
This table summarizes where your design has to be improved. Please make sure You stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

| Your Design |Figma Design|
| :---:   | :---: | 
|Check icon at the right|
| noicon | iconpresent |
| langauge list |lang-figmalist |
| arrowmiss |  arrowthere|
| nospace | spacepresent |

**Review by bienvenuushindi — APPROVED** [review]
# STATUS: APPROVED 🟢 

## Hi @ezraermy    👋, 

Well done implementing the requested changes 👏
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Highlights 🎯 
:+1: No linter errors✔️
:+1: Good PR title :heavy_check_mark: 

### Optional Suggestions
- **Please make sure you add all the necessary steps to set up your project in your README. As for now you just have headlines with no content**

- [ ] **For more suggestions  => Please check the inline comments below.** :arrow_down: 

**_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**